jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -661,13 +661,32 @@ $bMigrateProfile.Add_Click( {
             } else {
                 Write-ToLog "ConnectKey is populated, JumpCloud agent will be installed"
             }
-            $testResult, $userID = Test-JumpCloudUsername -JumpCloudApiKey $tbJumpCloudAPIKey.Text -JumpCloudOrgID $lbl_orgId.Text -Username $tbJumpCloudUserName.Text -Prompt $true
+
+            $testResult, $JumpCloudUserId, $jcUsername, $JCSystemUsername = Test-JumpCloudUsername -JumpCloudApiKey $tbJumpCloudAPIKey.Text -JumpCloudOrgID $lbl_orgId.Text -Username $tbJumpCloudUserName.Text -Prompt $true
             if ($testResult) {
                 Write-ToLog "Matched $($tbJumpCloudUserName.Text) with user in the JumpCloud Console"
             } else {
                 Write-ToLog "$($tbJumpCloudUserName.Text) not found in the JumpCloud console"
                 return
             }
+
+
+            if ($JCSystemUsername) {
+                Write-ToLog "User $($JCSystemUsername) has a local account on this system"
+                $wshell = New-Object -ComObject Wscript.Shell
+                $message = "Would you like to migrate the local user profile to the JumpCloud User $($jcsystemUserName)?"
+                $var = $wshell.Popup("$message", 0, "JC Local User Check", 64 + 4)
+                # If user selects yes then migrate the local user profile to the JumpCloud User
+
+                if ($var -eq 6) {
+                    Write-ToLog -Message "User selected Yes, continuing with migration $($jcsystemUserName))"
```

Comment:
```suggestion
                    Write-ToLog -Message "User selected Yes, continuing with migration of $($SelectedUserName) to $($jcsystemUserName)"
```

Reply:
![Screen Shot 2023-05-25 at 11 18 12 AM](https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/54448601/6f69a0bf-e1bb-43c2-a4f7-c06a821e2264)

This comment is a bit confusing, has an extra ')'. Let's change it a bit to say "User selected 'Yes', continuing with migration of "someUser" to "anotherUser"